src/tests/ftest/util/ior_utils.py

```diff
@@ -147,7 +147,7 @@ def set_daos_pool_params(self, pool, display=True):
             display (bool, optional): print updated params. Defaults to True.
         """
         self.daos_pool.update(
-            pool.pool.get_uuid_str(), "daos_pool" if display else None)
+            pool.get_uuid_str(), "daos_pool" if display else None)
```

Comment:
This change will break other tests that use `ior_utils.py` and call `set_daos_params()` with a `pool` argument that is a `TestPool` type, for example:
- container_create.py
- metadata.py
- ior_test_base,py
- mdtest_test_base.py

Reply:
ior_test_base.py and mdtest_test_base.py did ran fine as part of CI run but to address the other two I have moved from using DaosPool to TestPool.